test/scripts/update-param-sys.py

```diff
@@ -0,0 +1,97 @@
+########################################################################
+# This script is designed to function with OpenC3 COSMOS's Script Runner
+########################################################################
+from openc3.script import *
+import struct
+
+# Clear counts and data products from prior tests:
+cmd("Linux_Example Ccsds_Command_Depacketizer_Instance-Reset_Counts")
+cmd("Linux_Example Command_Router_Instance-Reset_Data_Products")
+wait(1.0)
+
+# Send Noop_Arg command expecting success:
+cmd("Linux_Example Command_Router_Instance-Noop_Arg with Value 1")
+wait(1.0)
+# Check successful command count is 2 and that the last success was Noop_Arg with last Value 1:
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Command_Success_Count.Value == '2'")
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Last_Successful_Command.Id == '3'")
+print("Noop_Arg success OK")
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Noop_Arg_Last_Value.Value == '1'")
+print("Noop_Arg last Value OK")
+
+# Send Noop command expecting failure:
+cmd("Linux_Example Command_Router_Instance-Noop_Arg with Value 868")
+wait(1.0)
+# Check last failed command count was 1 and that it was Noop_Arg with last Value 1:
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Command_Failure_Count.Value == '1'")
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Last_Failed_Command.ID == '3'")
+print("Noop_Arg failure OK")
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Noop_Arg_Last_Value.Value == '868'")
+check_formatted("Linux_Example Software_Status_Packet Command_Router_Instance.Last_Failed_Command.Status == 'FAILURE'")
+print("Noop_Arg last Value failure OK")
+
+#########################################################################
+# If this point was reached then begin testing parameter update functions
+#########################################################################
+
+# Build a parameter table using bytearrays:
+# Osc_A_Freq
+Test_Packed_Table = bytearray(struct.pack(">f", 0.30))
+# Osc_A_Amp
+Test_Packed_Table += bytearray(struct.pack(">f", 5.00))
+# Osc_A_Off
+Test_Packed_Table += bytearray(struct.pack(">f", 2.50))
+# Osc_B_Freq
+Test_Packed_Table += bytearray(struct.pack(">f", 0.30))
+# Osc_B_Amp
+Test_Packed_Table += bytearray(struct.pack(">f", -5.00))
+# Osc_B_Off
+Test_Packed_Table += bytearray(struct.pack(">f", -2.50))
+# Version
+Test_Packed_Table += bytearray(struct.pack(">f", 0.0))
+# Crc_Table
+Test_Packed_Table += bytearray(struct.pack(">h", 0))
+# Buffer_Length
+Test_Packed_Table += bytearray(struct.pack(">h", 55))
```

Comment:
Very cool that this works. But this is brittle in that your pattern here must match the parameter table or it will fail in hard-to-debug way. We should use the Adamant python autocoder to help here.

Reply:
Started testing creating the table records using the Adamant python autocoder in [8bc9f2d](https://github.com/lasp/adamant_example/pull/33/commits/8bc9f2db6bba36143c7866aaa371b33ed8c0494c).
This requires building and manually copying `linux_example_parameter_table_record.py` as well as the dependencies: `base_classes/packed_type_base.py`, ` packed_F32.py`, and `parameter_table_header.py`. These can be added to the COSMOS plugin setup script in a future commit.

Reply:
Setup script handles these files as of [42948d8](https://github.com/lasp/adamant_example/pull/33/commits/42948d8cea797db68d31707beb117741cd022fcc).